lib/registries/addon/components/hero-overlay/styles.scss

```diff
@@ -0,0 +1,43 @@
+.HeroOverlay {
```

Comment:
nice, the overlay looks great. i wonder if the background should have `filter: grayscale(1)` -- otherwise color may come through in a way that looks off (and ideally this is resilient to poor image choices with clashing colors)

with grayscale:
<img width="688" alt="Screen Shot 2020-03-26 at 13 59 10" src="https://user-images.githubusercontent.com/6776190/77680391-41d64780-6f6a-11ea-9fcb-c9cb523219b9.png">
without grayscale:
<img width="688" alt="Screen Shot 2020-03-26 at 13 58 52" src="https://user-images.githubusercontent.com/6776190/77680368-3b47d000-6f6a-11ea-9eb9-d8603bade0f3.png">

Reply:
I see what you mean. The brand primary color could also make it ugly. How about using grayscale, but using a value like .8 to tone it down but still retain some of the primary color?

Reply:
(Maybe something that can be discussed tomorrow during the design review)

Reply:
hm i'm not sure i follow -- i think the image (on `::before`) should be grayscaled all the way, but the overlay with `background-color: var(--primary-color)` (on `::after`) should be left as it is (partly transparent), so the primary color is what's dominant, if that makes sense